# Rover Dimension in [mm]
rover_length: 436.0
rover_width: 475.7
rover_height: 452.0

# Initial Camera Position in [mm]
# (Guessed values)
camera_headX: 45.0
camera_headY: 0.0
camera_headZ: 30.0
# Camera Offset from Pitch Axis in [mm]
# Stereo Camera (Guessed values)
stereo_cameraO: 0.0
# IR Camera (Guessed values)
ir_cameraO: 5.0

# IMU Position in [mm] and Orientation Quaternion
# Central IMU
c_imuX: 1.0
c_imuY: 0.0
c_imuZ: 0.0
#c_imuQX: 0.0
#c_imuQY: 0.0
#c_imuQZ: 0.0
#c_imuQW: 1.0

# Locomotion Control Parameters
# Driving
drive_enc_ppr: 8384                     # Drive encoder pulses per revolution
max_vel: 3.14159265358979323846         # Maximal wheel velocity [rad/s]
# Steering
steer_enc_ppr: 8384                     # Steer encoder pulses per revolution
max_ort: 1.57079632679489661923         # Maximal wheel orientation [rad]

# Elecrical Power Supply Parameters
# Current Sensors
# Set EPS parameters
max_cur_b: 25                           # Maximal current on battery current sensor [A]
max_cur_m: 12                           # Maximal current on motor current sensor [A]